### DISCUSSION TITLE
Can we provide snapshots of calibraion network and Nerpa network?

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/lotus#6938] [Ideas]
I can see that some people in the Chinese community are already doing it. The two networks run for a long time and sometimes need to be synchronized for a long time, but foreign people generally can't understand Chinese. If we provide snapshots and do relevant tutorials, it should be helpful to them

**Comment by ribasushi** [comment]
In order to download a snapshot produced by another party you just need a link. A language barrier shouldn't be a factor... no?

In any case: I much rather involve the community more and have lower-stakes snapshots be produced and distributed without PL's involvement. @dayou5168 how do we make this happen?

/cc @f8-ptrk

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
the only thing node operators will miss is the proper bandwidth to cater snapshots to a broad audience. we can provide snapshots of all networks if people really need it but we will not have the bandwidth or CDN to properly distribute them (nor will we touch anything cloudy...) 

if you can pull them and distribute them we are happy to make them once a day or so for nerpa and calib. i doubt that it needs more then daily

**Reply by ribasushi** [reply]
> the proper bandwidth to cater snapshots to a broad audience

This is **precisely** what I want the to be figured out going forward. Relying on PL for everything is not healthy.

The nerpa snapshot is less than a gigabyte compressed and takes **under one minute** to produce
```
~$ time lotus chain export --recent-stateroots 1802 --skip-old-msgs /dev/stdout \
| zstd -8 --long -T0 \
| tee >( wc -c 1>&2 ) \
| ipfs add --progress=false --cid-version=1 --wrap-with-directory --stdin-name=nerpanet_minimal_snapshot_2021-08-05.car.zst
770073471
added bafybeiwillnotmakethiseasyforyounopenopenopenope nerpanet_minimal_snapshot_2021-08-05.car.zst
added bafybeistillwillnotmakethiseasyforyoustillnopeno

real	0m49.022s
user	2m39.682s
sys	0m5.733s
```

This community is in the file-storage business: let's get it figured out!
